internal/storage/db/internal/db.go

```diff
@@ -38,6 +38,7 @@ type DBStorage interface {
 	ListPolicyIDs(ctx context.Context) ([]string, error)
 	ListSchemaIDs(ctx context.Context) ([]string, error)
 	AddOrUpdateSchema(ctx context.Context, schemas ...*schemav1.Schema) error
+	EnableOrDisable(ctx context.Context, disabled bool, policyKey ...string) error
```

Comment:
Is the "enable" portion ever used? That would be taken care of by `AddOrUpdate` anyway, wouldn't it? So, I would just call this `Disable`